cirq-core/cirq/devices/grid_qubit_test.py

```diff
@@ -393,22 +393,29 @@ def test_complex():
     assert isinstance(complex(cirq.GridQubit(row=1, col=2)), complex)
 
 
-def test_numpy_index():
-    np5, np6, np3 = [np.int64(i) for i in [5, 6, 3]]
+@pytest.mark.parametrize('dtype', (np.int8, np.int64, float, np.float64))
+def test_numpy_index(dtype):
+    np5, np6, np3 = [dtype(i) for i in [5, 6, 3]]
     q = cirq.GridQubit(np5, np6)
     hash(q)  # doesn't throw
```

Comment:
Can we require `assert hash(q) == hash(cirq.GridQubit(5, 6))` ?

And likewise at lines 407, 417 below.

Reply:
Good idea. Added.